cou/steps/analyze.py

```diff
@@ -157,9 +159,22 @@ def current_cloud_os_release(self) -> Optional[OpenStackRelease]:
         This property just consider OpenStack charms as those that have
         openstack-origin or source on the charm configuration (app.os_origin).
         :return: OpenStack release codename
-        :rtype: OpenStackRelease
+        :rtype: Optional[OpenStackRelease]
         """
         return min(
             (app.current_os_release for app in self.apps_control_plane + self.apps_data_plane),
             default=None,
         )
+
+    @property
+    def next_cloud_os_release(self) -> Optional[OpenStackRelease]:
+        """Shows the next OpenStack release codename.
+
+        This property just consider OpenStack charms as those that have
+        openstack-origin or source on the charm configuration (app.os_origin).
```

Comment:
This description it's not true anymore. We are checking openstack releases by channel on subordinates charms and by workload version on auxiliary charms. Please remove on `current_cloud_os_release` and `next_cloud_os_release`